src/makeInfinityAdder.js

```diff
@@ -4,7 +4,20 @@
  * @return {function}
```

Comment:
The JSDoc comment is missing a description for the parameter `number` in the returned function `adder`. Consider adding `@param {number} [number]` to document the parameter.